File: Frontend/src/app/models/usuario.ts
export class Usuario {
    id_user: string;
    nom_user: string;
    a_pat_user: string;
    a_mat_user: string;
    correo_user: string;
    contra_user: string;
    cel_user: string;
    direc_user: string;

    constructor(id:string, nom:string, a_pat:string, a_mat:string, correo:string,
        contra:string, cel:string, direccion:string){
            this.id_user = id;
            this.nom_user = nom;
            this.a_pat_user = a_pat;
            this.a_mat_user = a_mat;
            this.correo_user = correo;
            this.contra_user = contra;
            this.cel_user = cel;
            this.direc_user = direccion;
        }
}

File: Backend/src/models/usuario.py
from ..database.db import DatabaseManager

db = DatabaseManager().getInstancia()

class Usuario(db.db.Model):
    idUser      = db.db.Column(db.db.Integer, primary_key=True)
    nomUser     = db.db.Column(db.db.String(80))
    aPatUser    = db.db.Column(db.db.String(80))
    aMatUser    = db.db.Column(db.db.String(80))
    correoUser  = db.db.Column(db.db.String(80))
    contraUser  = db.db.Column(db.db.Text)
    celUser     = db.db.Column(db.db.Integer)
    direccion   = db.db.Column(db.db.String(160))

    def __init__(self, nom, aPat, aMat, correo, contra, cel, direccion):
        self.nomUser    = nom
        self.aPatUser   = aPat
        self.aMatUser   = aMat
        self.correoUser = correo
        self.contraUser = contra
        self.celUser    = cel
        self.direccion  = direccion
    
    def to_json(self):
        return {
            'id_user'       : self.idUser,
            'nom_user'      : self.nomUser,
            'a_pat_user'    : self.aPatUser,
            'a_mat_user'    : self.aMatUser,
            'correo_user'   : self.correoUser,
            'contra_user'   : self.contraUser,
            'cel_user'      : self.celUser,
            'direc_user'    : self.direccion
        }